SDF-to-Coords

```diff
@@ -1,15 +1,19 @@
-#Folder pathway: D:\SDFs\structures
-#This next bit of code is to see whether I can pull a file from the directory
-
-def SDF_to_coords(g):
-    path = "/mnt/d/SDFs/"
-    for g in path:
-        print(g)
-        
-g = "Aletamine.sdf"        
-SDF_to_coords(g)
-
-#Prints out "/mnt/d/SDFs/"
+import os
+
+import numpy as np
+
+def SDF_to_Coords(filename, num_atom):
+    directory = r'/mnt/d/SDFs/structures/'
+    for filename in os.listdir(directory):
+        if filename.endswith(".sdf"):
+            x = np.asarray(filename)
```

Comment:
Unless you've manually modified these files you're going to need to do more parsing than that. You'll have to look at a couple of the SDF files and see what the format is, but say it's always:three lines, the xyz coordinates, and then a blank line you could do something like (untested code):
```.py
xyz = []
with open(filename) as f:
    counter = 0
    for line in f:
         if counter < 3: # Ignore the first three lines
             counter += 1
             continue
         # We know we're past the third line now

         if(line.strip()): # Exit out of loop when we hit the blank line
              break
         
         # This line is part of the coordinates grab the first four columns
         xyz.append(line.split(' ')[:4]) 
```

Reply:
I could be wrong though, if your `x` looks reasonable just disregard this whole post